_includes/manuals/1.10/1.2_release_notes.md

```diff
@@ -2,6 +2,40 @@
 
 ### Headline features
 
+#### Orchestration rebuilder
```

Comment:
orchestration is only mentioned internally in the code, I wonder if there is a better user facing name for it?